tools/get_logs.sh

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env bash
```

Comment:
I'd call this file as getEnv.sh or something similar, you gather much more info than only logs.

Reply:
Done